DragonFruit.Six.API/Processing/RawConvert.cs

```diff
@@ -118,15 +118,30 @@ public static Season ToSeason(this JObject jObject, string guid)
             return new Season
             {
                 Guid = guid,
-
+                TimeUpdated = DateTime.Parse(json.GetString(SeasonalRanked.TimeUpdated, DateTime.Now.ToString())),
                 SeasonId = json.GetByte(SeasonalRanked.Season),
-                Rank = json.GetUInt(SeasonalRanked.Rank),
-                MaxRank = json.GetUInt(SeasonalRanked.MaxRank),
+
+                Kills = json.GetUInt(SeasonalRanked.Kills),
+                Deaths = json.GetUInt(SeasonalRanked.Deaths),
+                KD = (json.GetFloat(SeasonalRanked.Kills) / json.GetFloat(SeasonalRanked.Deaths)),
```

Comment:
```suggestion
                KD = (json.GetFloat(SeasonalRanked.Kills, 1) / json.GetFloat(SeasonalRanked.Deaths, 1)),
```
They need to default to 1 if they are in a KD or WL calculation as otherwise we get a `NaN` result